lib/rubygems/package/tar_reader/entry.rb

```diff
@@ -117,6 +138,13 @@ def pos
     bytes_read
   end
 
+  ##
+  # Seek to the position in the tar entry
+
+  def pos=(new_pos)
+    seek(new_pos, IO::SEEK_SET)
```

Comment:
Sholdn't this return `new_pos` similar `IO#seek` return value?

Reply:
That seems right for `#pos=`. I couldn't get seek to return anything but 0 no matter what I did.

Reply:
Clear, `seek` returns 0 constantly as expected. I was thinking about making it explicit to keep it clear, so later if `pos=` could be potentially changed, `return new_pos` will not depend on the of implementation. Anyway I'm ok to keep it as is if that seems simpler for you @martinemde.

Reply:
For reference: 
```ruby
irb(main):002:0> f = File.open('README.md')
=> #<File:README.md>
irb(main):003:0> f.seek(10)
=> 0
irb(main):004:0> f.pos
=> 10
irb(main):005:0> f.pos = 20
=> 20
irb(main):006:0> f.close
=> nil
```